openstack/manila-nanny/templates/deployment.yaml

```diff
@@ -9,7 +9,10 @@ metadata:
     system: openstack
     ccloud/support-group: compute-storage-api
     ccloud/service: manila
-
+    {{- if .Values.vpa.set_main_container }}
+  annotations:
+    vpa-butler.cloud.sap/main-container: db-consistency-and-purge
+  {{- end }}
 spec:
```

Comment:
nanny consists of several worker container running a separate job, they are no differences between them in terms of performance requirements. I would remove the main-container annotation for manila-nanny.

Reply:
OK removed annotation from manila-nanny